docs/developers/sdk.md

```diff
@@ -1,5 +1,5 @@
 # SDK
-HermezJS is an open source SDK to interact with Hermez Rollup network.  It can be downloaded as an npm package `@hermeznetwork/hermezjs`.
+HermezJS is an open source SDK to interact with Hermez Rollup network.  It can be downloaded as an [npm package](https://www.npmjs.com/package/@hermeznetwork/hermezjs), o via [github](https://github.com/hermeznetwork/hermezjs).
```

Comment:
```suggestion
HermezJS is an open-source SDK to interact with Hermez Rollup network.  It can be downloaded as an [npm package](https://www.npmjs.com/package/@hermeznetwork/hermezjs), or via [github](https://github.com/hermeznetwork/hermezjs).
```